src/showcase/showcase-directive.js

```diff
@@ -16,6 +16,15 @@
                     chart.periodsOfDataToFetch(configService.getBitfluxStockAmount());
                     chart.proportionOfDataToDisplayByDefault(configService.getInitialBitfluxProportion());
 
+                    // If there's already a selection, run the chart and use that.
+                    // This occurs when the user selects a stock in compact view
+                    // and it expands.
+                    if (scope.selection && scope.selection()) {
+                        firstRun = false;
+                        chart.run(element[0].children[0]);
```

Comment:
The code in the watch only calls `run` for the first run -- does it need to be called every time here?